buildspec-release.yml

```diff
@@ -0,0 +1,155 @@
+version: 0.2
+
+env:
+  variables:
+    FRAMEWORK_VERSION: '1.1.0'
+    CPU_INSTANCE_TYPE: 'ml.c4.xlarge'
+    CPU_PY_VERSION: '2'
+    GPU_INSTANCE_TYPE: 'ml.p2.xlarge'
+    GPU_PY_VERSION: '3'
+    LOCAL_BASE_REPO: 'pytorch-base'
+    ECR_REPO: 'sagemaker-pytorch'
+    GITHUB_REPO: 'sagemaker-pytorch-container'
+    SETUP_FILE: 'setup_cmds.sh'
+    SETUP_CMDS: '#!/bin/bash\npip install --upgrade pip\npip install -U -e .\npip install -U -e .[test]'
+
+
+phases:
+  pre_build:
+    commands:
+      - start-dockerd
+      - ACCOUNT=$(aws sts get-caller-identity --query 'Account' --output text)
+      - PREPROD_IMAGE="$ACCOUNT.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$ECR_REPO"
+      - PR_NUM=$(echo $CODEBUILD_SOURCE_VERSION | grep -o '[0-9]\+')
+      - echo 'Pull request number:' $PR_NUM '. No value means this build is not from pull request.'
+
+  build:
+    commands:
+      # prepare the release
+      - git-release --prepare --min-version 1.2.0
+
+      # install
+      - pip3 install -U -e .
+      - pip3 install -U -e .[test]
```

Comment:
Do you need to install these dependencies here with pip3?

Reply:
I was following the buildspec.yml using pip3, but I'll ask Yang about it.

Reply:
Both python 2 and 3 versions of pip are installed in the new build system, so we need to specify pip3.